contracts/base/OwnerManager.sol

```diff
@@ -138,7 +139,7 @@ abstract contract OwnerManager is SelfAuthorized {
      * @return Boolean if owner is an owner of the Safe.
      */
     function isOwner(address owner) public view returns (bool) {
-        return owner != SENTINEL_OWNERS && owners[owner] != address(0);
+        return !(owner == SENTINEL_OWNERS || owners[owner] == address(0));
```

Comment:
Is this intentional? I'm surprised that this reduced codesize TBH (looks like an extra operation).

Reply:
Oh nice, I miscounted.